src/vs/workbench/api/common/extHostTextEditor.ts

```diff
@@ -566,6 +566,9 @@ export class ExtHostTextEditor {
 			},
 			hide() {
 				_proxy.$tryHideEditor(id);
+			},
+			[Symbol.for('debug.description')]() {
+				return `TextEditor(${this.document.uri.toString()}, ${this.selections.map(s => getDebugDescriptionOfSelection(s)).join(', ')})`;
```

Comment:
Unsure, including the selection can be huge and confusing. It makes me also wonder how the equivalent for notebook looks

Reply:
Lets remove the selection then.